src/org/jitsi/impl/neomedia/rtcp/FIRPacket.java

```diff
@@ -13,19 +13,27 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jitsi.service.neomedia.rtp;
-
-import org.jitsi.impl.neomedia.rtcp.*;
+package org.jitsi.impl.neomedia.rtcp;
 
 /**
- * A simple interface for handling RTCP NACK packets.
- * @author Boris Grozev
+ * @author George Politis
  */
-public interface NACKListener
+public class FIRPacket
+    extends RTCPFBPacket
 {
     /**
-     * Handles an RTCP NACK packet.
-     * @param nackPacket the packet.
+     * The FMT of an FIR packet.
+     */
+    public static final int FMT = 4;
+
+    /**
+     * Ctor.
+     *
+     * @param senderSSRC
+     * @param sourceSSRC
      */
-    public void nackReceived(NACKPacket nackPacket);
+    public FIRPacket(long senderSSRC, long sourceSSRC)
+    {
+        super(FMT, PSFB, senderSSRC, sourceSSRC);
```

Comment:
what does PSFB stand for?  looks like there isn't any comment at the definition.  might also be clearer to refer to it as RTCPFBPacket.PSFB (helps give some more context?)